rpc/walletrpc/api.pb.go

```diff
@@ -2555,7 +2555,7 @@ type TransactionNotificationsResponse struct {
 	// Any new unmined transactions are included here.  These unmined transactions
 	// refer to the current best chain, so transactions from detached blocks may
 	// be moved to mempool and included here if they are not mined or double spent
-	// in the new chain.  Additonally, if no new blocks were attached but a relevant
+	// in the new chain.  Additionally, if no new blocks were attached but a relevant
```

Comment:
this file is autogenerated so don't change this here.  Fix the misspelling in api.proto instead.